### ISSUE TITLE
Issue with dynamic template loading in Tabset

### ISSUE THREAD

**Issue by ssankarsiva** [mikejacobson/angular-bootstrap-scrolling-tabs#8]
Thanks for the awesome directive, As suggested in Use case 3, I have followed the implementation.

But I could not able to load the dynamic HTML content in the container.

I am using ng-include ("tabContainer" variable from angular to load HTML template file path) for loading the HTML content in container via angular.

Below are my snippet

HTML:

&#x3C;div scrolling-tabs-wrapper  watch-tabs=&#x22;tabList&#x22; scroll-to-tab-edge=&#x22;true&#x22;&#x3E;
&#x9;&#x3C;tabset&#x3E;
&#x9;&#x9;&#x3C;tab heading=&#x22;{{tabName}}&#x22; ng-click=&#x22;getTabPanel(tabName)&#x22; ng-repeat=&#x22;tabName in tabList&#x22;&#x3E;
&#x9;&#x9;&#x9;&#x3C;div class=&#x22;col-md-12 evenPadding&#x22; ng-include=&#x22;tabContainer&#x22;&#x3E;&#x3C;/div&#x3E;
&#x9;&#x9;&#x3C;/tab&#x3E;
&#x9;&#x3C;/tabset&#x3E;
&#x3C;/div&#x3E;

Error: $compile:ctreq
Missing Required Controller
Controller 'tabset', required by directive 'tabContentTransclude', can't be found!

Thanks
Siva

**Comment by mikejacobson** [comment]
Are you sure you have the latest code? That error was getting thrown before when tabs were added dynamically after page load using AngularUI tabset, but I checked in a fix for it.

Here's a plunk showing it working with the tab content loaded using `ng-include`:
http://plnkr.co/edit/WGeY2gyMYCMBTr4UzSzb?p=preview

It seems to be working fine. The 'col-md-12' class on the transcluded content seems to cause the content element to cover the tabs and make them unclickable (I removed that class in my plunk), but I assume that can be fixed via CSS.
